types/react/package.json

```diff
@@ -9,6 +9,16 @@
         "default": "./index.d.ts"
       }
     },
+    "./next": {
```

Comment:
This doesn't seem to have an equivalent in the react 18 source - minimally, it's not in their export map, so such an import will fail at runtime if it's not elided. What's even in this entrypoint?